### DISCUSSION TITLE
写了个kbengine的异步http server，欢迎吐槽

### DISCUSSION THREAD

**Discussion by worldg** [kbengine/kbengine#1350] [General]
使用了 kbengine提供的fd 的接口，借鉴 tornado 和 sanic 写了个http server 框架，功能还很简单，不过对于简单的httpserver接口够用了，可以用在支付回调中

在GitHub上面  https://github.com/asynchronoust/kbengine-httpserver

欢迎吐槽

**Comment by ybyaoyao** [comment]
谢谢！
